docs/sources/panels-visualizations/visualizations/flame-graph/index.md

```diff
@@ -18,55 +18,77 @@ weight: 850
 
 # Flame graph panel
```

Comment:
```suggestion
# Flame graph
```

Reply:
I added the panel here because next section is called `Flame graph`. The issue it the whole panel consists of two sections Table + Flame graph and so wanted to explain by sections, first Flame graph then Table. But that creates the problem of having repeated headers right next to each other.

Reply:
Ah I understand. This is tricky because ultimately, as we start work on making all of these Visualizations docs more consistent, we likely won't want the word "panel" in any of the page titles. What if we added the word "mode" to the two sub-headings: Flame graph mode, Top table mode?

Reply:
Good idea, yeah will change it.